Dockerfile

```diff
@@ -48,7 +48,8 @@ gpgcheck=1\n\
 gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub' \
  > /etc/yum.repos.d/google-chrome.repo \
   && yum install -y epel-release \
-  && yum install -y mysql-devel \
+  && yum install -y rh-mysql80-mysql-devel \
+                   rh-mysql80-mysql-config \
```

Comment:
that `rh-` makes me wonder if these packages require a subscription ... 

but I guess we'll find out when we try to build on circleci.